### ISSUE TITLE
Unable to import webpack plugin in ESM

### ISSUE THREAD

**Issue by sqal** [getsentry/sentry-javascript-bundler-plugins#274]
### Environment

@sentry/webpack-plugin: 2.1.0
node: 18.16.0

### Steps to Reproduce

Set `"type": "module"` in package.json and try to import the plugin with esm

```json
{
  "name": "my-app",
  "version": "0.0.0",
  "private": true,
  "type": "module", 
  "scripts": {
    "build": "node index.js"
  },
  "devDependencies": {
    "@sentry/webpack-plugin": "^2.1.0",
    "webpack": "^5.84.0"
  }
}
```

```js
// index.js
import { sentryWebpackPlugin } from "@sentry/webpack-plugin";
```

### Expected Result

I should be able to import the plugin in ES Modules.

### Actual Result

I cannot import the plugin in ES Modules, I see the following error:

```
file:///C:/Users/.../node-module/node_modules/@sentry/webpack-plugin/dist/esm/index.mjs:5
import { BannerPlugin } from 'webpack-4';
         ^^^^^^^^^^^^
SyntaxError: Named export 'BannerPlugin' not found. The requested module 'webpack-4' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'webpack-4';
const { BannerPlugin } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:190:5)
```

**Comment by lforst** [comment]
Hey, can you try the newest version we released 15 minutes ago (`2.2.0`). It's likely fixed there.

**Comment by lforst** [comment]
I totally anticipated your issue 😏
